gcn/notices/core/Reporter.schema.json

```diff
@@ -22,15 +22,28 @@
       "description": "Messenger of report; EM, GW or Neutrino"
     },
     "spectrum": {
-      "enum": ["energy", "wavelength", "frequency"],
-      "description": "high-energy or optical or radio observations, if not parsed, then default band is energy",
       "type": "string",
+      "enum": ["energy", "wavelength", "frequency"],
+      "description": "Mode of spectral measurement: high-energy or optical or radio observations, if not parsed, then default is energy",
```

Comment:
Perhaps this should just be simplified into:
"Mode of spectral measurement; default is energy"